autosklearn/experimental/askl2.py

```diff
@@ -339,6 +335,7 @@ def __init__(
             "classifier": include_estimators,
             "feature_preprocessor": include_preprocessors,
         }
+        train_selectors()
```

Comment:
Perhaps we can improve the startup time further by only training the selector for the `metric` that is given. In the case of no metric given, I guess we can just default to using all of them.

Reply:
I do see the `metric` parameter that is accepted by askl2. Should this be a list of metrics? And if nothing is passed, we could use the default ones.